integreat_cms/cms/urls/protected.py

```diff
@@ -429,7 +429,7 @@
             [
                 path("", include(media_ajax_urlpatterns)),
                 path(
-                    "locations/auto-complete-address/",
+                    "<region_slug>/locations/auto-complete-address/",
                     pois.auto_complete_address,
                     name="auto_complete_poi_address",
                 ),
```

Comment:
One small improvement for consistency could be to move this down to the other ajax endpoints that require a region slug. It could for example be grouped with `search-poi` at line 626.

Reply:
Thank you, done!